app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -1,90 +0,0 @@
-input,
-input[type=email],
-input[type=text],
-input[type=password],
-textarea { /* Bootstrap reset */
-  &,
-  &:active,
-  &:invalid,
-  &:invalid:required,
-  &:focus,
-  &:active:focus,
-  &:invalid:focus,
-  &:invalid:required:focus {
-    border-color: $border-grey;
-    box-shadow: none;
-    color : $text-dark-grey;
-  }
-}
-
-/* autocomplete colors */
-input:-webkit-autofill{
-  background-color: #fff !important;
-  background-image: none !important;
-}
```

Comment:
In Chromium the background was still yellow so I removed these lines and I don't think that we have to override the background.